foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePDPPythonMigrationSteps.java

```diff
@@ -0,0 +1,76 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+
+import static org.junit.Assert.assertTrue;
+
+public class AiopsReferencePDPPythonMigrationSteps extends AbstractMigrationTest {
+
+    private AbstractAissembleMigration migration;
+    private File validatedFile;
+    private static Boolean validateMigration(File original, File migrated) {
+        Boolean isMigrated = false;
+
+        try {
+            isMigrated =  FileUtils.contentEquals(original, migrated);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return isMigrated;
+    }
+
+    /**
+     * Function to check whether the migration is necessary.
+     * @param file file to check
+     * @return isFileNameSame - Boolean indicating whether the file names are the same.
+     */
+
+    public static Boolean isAissembleSecurityProperties(File file) {
+        Boolean isFileNameSame = file.getName().equals("aissemble-security.properties");
+        return isFileNameSame;
+    }
+
+    @Given("a Python implemented project policy decision point docker properties file is referencing aiops")
+    public void aPythonImplementedProjectPolicyDecisionPointDockerPropertiesFileIsReferencingAiops() {
+        testFile = getTestFile("v1_8_0/AiopsReferencePDPPythonMigration/migration/aiops-security.properties");
+    }
+
+    @When("the 1.8.0 aiops reference pdp python migration executes")
+    public void theAiopsReferencePdpPythonMigrationExecutes() {
+        migration = new AiopsReferencePDPPythonMigration();
+        performMigration(migration);
+    }
+
+    @Then("the properties file will be renamed to aissemble-security.properties")
+    public void thePropertiesFileWillBeRenamedToAissembleSecurityProperties() {
+        assertTrue("Python modules are referencing an older package in imports.", isAissembleSecurityProperties(getTestFile("v1_8_0/AiopsReferencePDPPythonMigration/migration/aissemble-security.properties")));
```

Comment:
Should this assert message pertain to the file not being renamed correctly?

Reply:
Updated the assert message.